xds/internal/xdsclient/transport/transport.go

```diff
@@ -428,6 +438,7 @@ func (t *Transport) recv(stream adsStream) bool {
 	for {
 		resources, url, rVersion, nonce, err := t.recvAggregatedDiscoveryServiceResponse(stream)
 		if err != nil {
+			// todo(arvindbright): call ErrHandler iff no msgs were ever recv'd on the stream
```

Comment:
Nit: s/todo/TODO. The latter is usually highlighted by text editors.

Reply:
Actually how much more work is this? If this is not much, it would be nicer to do this as part of this PR so that we can see all the changes in one place.